telemetry/packages/fake/src/sensorClasses/sensorIndex.ts

```diff
@@ -0,0 +1,24 @@
+import { Sensor } from "../baseSensor";
+// individual sensor classes
+import { Motion } from "./motion";
+import { Keyence } from "./keyence";
+import { Pressure } from "./pressure";
+import { Temperature } from "./temperature";
+import { Resistance } from "./resistance";
+import { Magnetism } from "./magnetism";
+import { Levitation } from "./levitation";
+
+// Package all sensors into one object to be iterated through easily in SensorManager
+interface sensorObj<T extends Sensor> {
```

Comment:
Since this type isn't reused anywhere else, you're probably best annotating `sensors` below with `Record<string, typeof Sensor>` which would enforce the correct typing of that object

Reply:
Removed the redundant type def, although it has to be typeof Motion | ... | Levitation as they have different structure.